MmSupervisorPkg/Library/StandaloneMmHobLibSyscall/StandaloneMmHobLibSyscall.c

```diff
@@ -248,6 +249,15 @@ CreateHob (
 
   HandOffHob = GetHobList ();
 
+  // MU_SEC_TCBZ4166 [BEGIN] - mitigate potential integer overflow

```

Comment:
Is the same in the BASECORE already? If not, can we do that as well? We will need to update the override hash once BASECORE implementation is updated, so might want to coalesce the code and override update in one?

Reply:
I'm working on updating BASECORE right now

Reply:
Shouldn't that ultimately come from a edk2 patch? edk2 -[cherry-pick]->Mu Basecore->[override hash update]->Mu Feature MM Supv.